packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -86,6 +87,8 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
     };
   }, [searchParams, firstNameQueryParam, lastNameQueryParam]);
 
+  const prevResponse = JSON.parse(localStorage.getItem(`prevEventTypeResp`) ?? "{}");
```

Comment:
Should we store the info per event-type as well? i.e. if a user books using different names for different events, should we owner that? I think if we do want to do it we can do it later.

Better name could be lastBookingResponse as it is the response for any of the bookings made by the user across any of the organizers.

As the customer is looking for feature parity with Calendly here, can we test out their behaviour in terms of booking different events with different emails.

Reply:
Since it would only be name and email, I think storing it per event type would be a bit too much. But as you said in case the need arises, that would be better done in a follow up PR.

I checked out how calendly does it, and it also stores a common one across all event types.